src/flow-worker.c

```diff
@@ -281,6 +281,19 @@ static TmEcode FlowWorker(ThreadVars *tv, Packet *p, void *data, PacketQueue *pr
     // Outputs.
     OutputLoggerLog(tv, p, fw->output_thread);
 
+    /* Prune any stored files. */
+    if (p->flow && p->flow->alstate) {
+        Flow *f = p->flow;
+        FileContainer *ffc_ts = AppLayerParserGetFiles(p->proto, f->alproto,
```

Comment:
is there a reason for first fetching both containers but then using only one? Packet is either toserver or toclient.

Reply:
I see this comes from the ``OutputFileLog``. Wonder why we did it this way.

Reply:
Looks like we added this in f75df19c35b6a36c00083ebaf7d06ecf4efac70e, but that is not relevant here I think.